spec/requests/users_spec.rb

```diff
@@ -49,4 +49,49 @@
       end
     end
   end
+
+  describe 'EDIT user profile' do
+    context 'when user is authenticated' do
+      let!(:user) { create(:user) }
+      let(:new_attributes) { { first_name: 'NewFirstName', last_name: 'NewLastName', current_password: user.password } }
```

Comment:
Remember that you can use the method attributes_for, which gives you a hash with attributes defined for an specific model based on its factory.